yoti-stub/src/services/YotiRequestProcessor.ts

```diff
@@ -46,26 +41,55 @@ export class YotiRequestProcessor {
 
 	async createSession(event: APIGatewayProxyEvent, yotiSessionItem: YotiSessionItem): Promise<Response> {
 
-		await this.yotiService.createYotiSession(yotiSessionItem);
- 
-		return new Response(HttpCodesEnum.CREATED, JSON.stringify(yotiSessionItem));
+		const sessionId = yotiSessionItem.session_id;
+		console.log("SESSION ID: ", sessionId);
+		const lastUuidChars = sessionId.slice(-4);
 
+		if (lastUuidChars[0] === '3' || lastUuidChars[0] === '2') {
+				console.log(JSON.stringify(new YotiSessionRequest(sessionId)));
+				// CREATE_SESSION.user_tracking_id = sessionId;
+				return new Response(HttpCodesEnum.CREATED, JSON.stringify(sessionId));	
```

Comment:
think we need to return the success response body too for these with `sessionId` same as `user_tracking_id`